README.md

```diff
@@ -247,7 +249,7 @@ You should see something similar to this:
    0.77369%   100000 of 12925066   (12282M free)
 ```
 
-If you see lots of red error messages, something went wrong. You can ask for help debugging it in the witness channel on [STEEMIT.CHAT](https://steemit.chat/channel/witness).
+If you see lots of red error messages, something went wrong. You can ask for help debugging it on the witness channel on [STEEMIT.CHAT](https://steemit.chat/channel/witness).
```

Comment:
"in" is already correct. When referring to an online chat, "in" a chat is the best description.

Reply:
Agreed.